ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -631,7 +669,8 @@ private void prepareForSending() {
     void onFillComplete() {
       buf.flip();
       prepareForSending();
-      chunkReadyTimeMs = time.milliseconds();
+      chunkReadyAtMs = time.milliseconds();
+      routerMetrics.chunkFillTimeMs.update(chunkReadyAtMs - chunkFreeAtMs);
```

Comment:
Is this actually the time to fill the chunk? What if the chunk was free but the channel did not have data? 

Other than possibly changing this, do you think it useful to track how long you wait for the channel to give you data (or is already done)?